server/server.go

```diff
@@ -0,0 +1,101 @@
+package server
+
+import (
+	"context"
+	"fmt"
+	"io"
+	"net/http"
+	"os"
+	"time"
+
+	"github.com/brave-intl/bat-go/middleware"
+	"github.com/brave/go-sync/controller"
+	"github.com/brave/go-sync/datastore"
+	"github.com/getsentry/sentry-go"
+	"github.com/go-chi/chi"
+	chiware "github.com/go-chi/chi/middleware"
+	"github.com/rs/zerolog"
+	"github.com/rs/zerolog/diode"
+	"github.com/rs/zerolog/hlog"
+	"github.com/rs/zerolog/log"
+)
+
+func setupLogger(ctx context.Context) (context.Context, *zerolog.Logger) {
```

Comment:
use `github.com/brave-intl/bat-go/utils/logging/SetupLogger`

Reply:
Addressed in https://github.com/brave/go-sync/pull/1/commits/381815052e300e4071343ff48c0608f665ee4673